sqldelight-runtime/src/commonMain/kotlin/com/squareup/sqldelight/logs/LogSqliteDriver.kt

```diff
@@ -0,0 +1,80 @@
+package com.squareup.sqldelight.logs
+
+import com.squareup.sqldelight.Transacter
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.db.SqlPreparedStatement
+
+class LogSqliteDriver(
+        private val sqlDriver: SqlDriver,
+        private val logger: (String) -> Unit
+): SqlDriver {
+
+    override fun currentTransaction(): Transacter.Transaction? {
+        return sqlDriver.currentTransaction()
+    }
+
+    override fun execute(identifier: Int?, sql: String, parameters: Int, binders: (SqlPreparedStatement.() -> Unit)?) {
+        log("EXECUTE\n $sql")
+        logParameters(binders)
+        sqlDriver.execute(identifier, sql, parameters, binders)
+    }
+
+    override fun executeQuery(identifier: Int?, sql: String, parameters: Int, binders: (SqlPreparedStatement.() -> Unit)?): SqlCursor {
+        log("QUERY\n $sql")
+        logParameters(binders)
+        return sqlDriver.executeQuery(identifier, sql, parameters, binders)
+    }
+
+    override fun newTransaction(): Transacter.Transaction {
+        log("TRANSACTION BEGIN")
```

Comment:
can you also add a unit test for this class which covers all the different logs that can appear?

Reply:
I was thinking about unit-tests, but I'm not sure where I should put them. I was unit testing it with AndroidSqliteDriver, but this package doesn't seem appropriate

Reply:
easiest thing would be to have it in commonTest and to create a `FakeDriver` that implements the `SqlDriver` interface but doesnt really do anything, that way you dont need to use an actual SqlDriver/instance to test the logging.